docs/serving/knative-kubernetes-services.md

```diff
@@ -5,25 +5,36 @@ weight: 9
 type: "docs"
 ---
 
-This document describes what is running when running knative serving.
+This guide describes the components that are active when running Knative Serving.
 
-After applying the serving yaml, this will install a few knative services and
-deployments on your kubernetes cluster. This document provides an overview of
-the deployments and the motivations for each one.
+## Before You Begin
```

Comment:
Good instincts here with showing users how they'd even get a service running in the first place! But I'd recommend doing this slightly differently. Since this is conceptual content, I'd recommend saying something like, "This guide assumes you already have a Knative Service running. If you don't, see Getting Started With App Deplyoment[link]." And then remove the steps you added. That way we're not duplicating content that exists eleswhere, and this page can have a strong focus on explaining what the various running services are.